snaplet/seed/define-seed.ts

```diff
@@ -69,6 +117,47 @@ export const seedUsers = async (length?: number) => {
       }),
     );
   }
+  let counter = 1;
+  const addresses = Array.from({ length }).map(() => {
```

Comment:
o segundo argumento do callback da função `map` é o index do elemento da array, não precisaria desse `counter` utilizando ele
```suggestion
  const addresses = Array.from({ length }).map((_, index) => {
```